src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -2580,126 +2584,6 @@
                 "skipUrlSync": false,
                 "type": "datasource"
             },
-            {
```

Comment:
QQ why was this removed

Reply:
This is automatically generated by cos suite (+ here it had the wrong definition which made the dashboard fail in multiple situations.